docs/contributing/contributing-code/contributing-code-control-plane/running-controlplane-locally.md

```diff
@@ -105,17 +96,73 @@ Run this command to create the namespace that will be used to store data.
 kubectl create namespace radius-testing
 ```
 
-## Setup Step 3: Create Resource Group and Environment
+## Setup Step 4: Setup Deployment Engine 
+
+
+> 💡 The Bicep deployment engine uses .NET. However you don't need to know C# or .NET to develop locally with Radius.
```

Comment:
`The Bicep deployment engine uses .NET. However you don't need to know C# or .NET to develop locally with Radius.`

Should this be in step 5 where we are asking users to install .NET?